website/docs/docs/on-premises/docker-compose-example.md

```diff
@@ -56,9 +56,10 @@ version: "3"
 
 services:
   pactflow:
-    image: quay.io/pactflow/enterprise
+    image: quay.io/pactflow/enterprise:2.0.0
```

Comment:
Should we suggest they just use the `:2` tag (if available)? Otherwise, these docs will become outdated very quickly

Reply:
Or perhaps just `latest`?